src/lpc.jl

```diff
@@ -21,13 +22,13 @@ function implements the mathematics published in [1].
 (DAFX 2003 article, Lagrange et al)
 http://www.sylvain-marchand.info/Publications/dafx03.pdf
 """
-function lpc{T <: Number}(x::AbstractVector{T}, p::Int, ::LPCBurg)
+function arburg{T <: Number}(x::AbstractVector{T}, p::Int)
     ef = x                      # forward error
     eb = x                      # backwards error
     a = [1; zeros(T, p)]        # prediction coefficients
-
     # Initialize prediction error wtih the variance of the signal
-    prediction_err = (sum(abs2, x) ./ length(x))[1]
+    prediction_err :: T = sum(abs2, x) ./ length(x)
```

Comment:
no, change it to / if you see fit. 
This was not changed in the current commit, I probably wrote it as to be explicit about doing a scalar operation (rather than a broadcast)